data/brands/amenity/pharmacy.json

```
@@ -2515,8 +2515,81 @@
         "amenity": "pharmacy",
         "brand": "Super-Pharm",
         "brand:wikidata": "Q13220217",
+        "brand:wikipedia": "en:Super-Pharm",
         "healthcare": "pharmacy",
-        "name": "Super-Pharm"
+        "name": "Super-Pharm",
+        "name:he": "סופר-פארם"
+      }
+    },
+    {
+      "displayName": "מאוחדת פארם",
+      "locationSet": {
+        "include": ["il"]
+      },
+      "tags": {
+        "amenity": "pharmacy",
+        "brand": "מאוחדת פארם",
```

```suggestion
        "brand": "מאוחדת פארם",
        "brand:en": "Meuhedet Pharm",
        "brand:he": "מאוחדת פארם",
```

If possible, can you duplicate name:he? because this can avoid wrong fallback to English on Hebrew language computer.

See: https://wiki.openstreetmap.org/wiki/Multilingual_names#Repeating_name_with_language_specific_tag

---

Sure, fixed